hypersync-net-types/src/bloom_filter_wrapper.rs

```diff
@@ -0,0 +1,97 @@
+use hex::{decode, encode};
```

Comment:
should use faster-hex and maybe even prefixed hex encoding like the functions in `format` crate

Reply:
Moved it into format and using the encoding/decoding from `Data`